packages/web/src/migrations/update-15-5-3/update-babel-preset.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  readJson,
+  writeJson,
+  Tree,
+  addDependenciesToPackageJson,
+  getProjects,
+  joinPathFragments,
+} from '@nrwl/devkit';
+import { nxVersion } from '../../utils/versions';
+
+/* Updates @nrwl/web/babel to @nrwl/js/babel because web package is no longer necessary to use webpack/rollup + babel. */
+export default async function update(tree: Tree) {
+  // Add `@nrwl/js` in case it was missing before.
+  addDependenciesToPackageJson(
+    tree,
+    {},
+    {
+      '@nrwl/js': nxVersion,
+    }
+  );
+
+  const projects = getProjects(tree);
+
+  projects.forEach((config, name) => {
+    const babelrcPath = joinPathFragments(config.root, '.babelrc');
+
+    if (!tree.exists(babelrcPath)) return;
+
+    const babelrc = readJson(tree, babelrcPath);
+    const idx = babelrc?.presets?.findIndex((p) =>
+      typeof p === 'string'
+        ? p === '@nrwl/web/babel'
+        : p[0] === '@nrwl/web/babel'
```

Comment:
Is this safe to assume, or better do `includes` in the array of `p`?

Reply:
yeah, it's always the first item in the array for bablrc files.